pkgs/tools/text/difftastic/default.nix

```diff
@@ -1,56 +1,36 @@
 { lib
-, fetchpatch
 , rustPlatform
 , fetchFromGitHub
 , testers
 , difftastic
-, stdenv
 }:
 
-let
-  mimallocPatch = fetchpatch {
-    name = "fix-build-on-older-macos-releases.patch";
-    url = "https://github.com/microsoft/mimalloc/commit/40e0507a5959ee218f308d33aec212c3ebeef3bb.patch";
-    sha256 = "sha256-DK0LqsVXXiEVQSQCxZ5jyZMg0UJJx9a/WxzCroYSHZc=";
-  };
-in
-
 rustPlatform.buildRustPackage rec {
   pname = "difftastic";
-  version = "0.56.1";
+  version = "0.58.0";
 
   src = fetchFromGitHub {
     owner = "wilfred";
     repo = pname;
     rev = version;
-    hash = "sha256-XQzsLowHtgXIKfUWx1Sj1D0F8scb7fNp33Cwfh5XvJI=";
+    hash = "sha256-PTc8/NhWsLcKJj+9ebV/YaWEmyOWKJCYUjmVbr4z2SY=";
   };
 
-  cargoLock = {
-    lockFile = ./Cargo.lock;
-    outputHashes = {
-      "tree_magic_mini-3.0.2" = "sha256-iIX/DeDbquObDPOx/pctVFN4R8GSkD9bPNkNgOLdUJs=";
-    };
-  };
+  cargoSha256 = "sha256-wtWJ32bBoCmx+eGkrOk8o0OQFMhGYZEBEDI1oHn3Zuw=";
```

Comment:
We use `cargoHash` now.

Reply:
Feel free to PR a fix.

Reply:
Feel free to avoid merging without having other maintainer's feedback ;)